pywbem_mock/_mainprovider.py

```diff
@@ -733,50 +719,43 @@ def add_namespace(self, namespace):
         except ValueError:
             raise CIMError(
                 CIM_ERR_ALREADY_EXISTS,
-                _format("Namespace {0!A} already exists in the mock "
-                        "repository", namespace))
+                _format("Namespace {0!A} already exists in the datastore ",
+                        namespace))
 
     def remove_namespace(self, namespace):
         """
-        Remove a CIM namespace from the mock repository.
+        Remove a CIM namespace from the datastore.
 
-        The namespace must exist in the mock repository and must be empty.
-
-        The default connection namespace cannot be removed.
+        The namespace must exist in the datastore and must be empty.
 
         Parameters:
 
           namespace (:term:`string`):
-            The name of the CIM namespace in the mock repository. Must not be
+            The name of the CIM namespace in the datastore. Must not be
             `None`. Any leading and trailing slash characters are split off
             from the provided string.
```

Comment:
Talking about splitting them off is appropriate for add_namespace() but is not optimal when the parameter is just used for looking up a namespace, like here. Use the text that you end up using in `validate_namespace()`, for consistency.

Reply:
Let's fix that later. ok for now.

Reply:
I refactored all the documentation of the namespace parameter to one standard set of phrases per later comment   **DONE**